spark/src/main/spark-3.x/org/apache/spark/sql/comet/shims/ShimCometScanExec.scala

```diff
@@ -46,25 +41,14 @@ trait ShimCometScanExec {
     .map { a => a.setAccessible(true); a }
     .flatMap(_.invoke(wrapped).asInstanceOf[Seq[AttributeReference]])
 
-  // TODO: remove after dropping Spark 3.2 and 3.3 support and directly call
+  // TODO: remove after dropping Spark 3.3 support and directly call
```

Comment:
nit `metadataColumns` is still needed for Spark 3.3 but we can update the comment